tests/jlm/llvm/frontend/llvm/MemCpyTests.cpp

```diff
@@ -25,7 +25,8 @@ MemCpyConversion()
   std::unique_ptr<Module> llvmModule(new Module("module", context));
 
   auto int64Type = Type::getInt64Ty(context);
-  auto pointerType = Type::getInt64PtrTy(context);
```

Comment:
Same as above

Reply:
same as above

Reply:
Yes, but it is still the one with the type argument, and not the one without.